scipy/optimize/_removeRedundancy.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Feb 25 22:08:23 2017
+
+@author: Matt
```

Comment:
These look like they should be `# comments` not initial docstring.
It's not necessary to record the creation date etc editing details (version control records that).

File name should be `_remove_redundancy.py`.